src-schema/Migration.php

```diff
@@ -167,10 +168,16 @@ public function field(string $fieldName, $options = []): self
             $column->setNotnull(false);
         }
 
-        if ($column->getType() === 'integer' && $refType !== self::REF_TYPE_NONE) {
+        if ($column->getType()->getName() === 'integer' && $refType !== self::REF_TYPE_NONE) {
             $column->setUnsigned(true);
         }
 
+        if (in_array($column->getType()->getName(), ['string', 'text'], true)) {
+            if ($this->getDatabasePlatform() instanceof SqlitePlatform) {
+                $column->setPlatformOption('collation', 'NOCASE');
```

Comment:
standard behaviour os MySQL string/text column (but only ASCII support)

small test will be good, if anyone want to contribute ;-)